languages/csharp/config.json

```diff
@@ -178,6 +196,18 @@
         "concepts": ["strings"],
         "prerequisites": ["basics"]
       },
+      {
+        "slug": "structs",
```

Comment:
I think this should be part of another PR?

Reply:
as for `casting`

Reply:
Okay. Keep it in.